Makefile

```diff
@@ -88,5 +88,5 @@ coverage-report-md:
 venv: requirements-optional.txt requirements.txt
 	python3 -m venv ${VENV_DIR}
 	source ${VENV_DIR}/bin/activate
-	pip install -r requirements-optional.txt -r requirements.txt
-	pip install -e ".[extras]"
+	python3 -m pip --require-virtualenv install -r requirements-optional.txt -r requirements.txt
```

Comment:
why the python3?

Reply:
Just be consistent with line 89 `python3 -m venv ${VENV_DIR}`

Reply:
ah didn't spot that... @AntonLydike is that so people don't run it with Python 2 or Python 4? :)

Reply:
Just a weird habit I picked up a while back. Maybe related to some work on ancient servers with questionable defaults^^